FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Multitask/DemoTasks/MultitaskMQTTExample.c

```diff
@@ -1779,12 +1944,14 @@ void prvSubscribeTask( void * pvParameters )
          * the last publish. */
         if( ++ulWaitCounter > mqttexampleMAX_WAIT_ITERATIONS )
         {
-            LogError( ( "Publish receive loop exceeded maximum wait time." ) );
+            LogError( ( "Publish receive loop exceeded maximum wait time.\n" ) );
```

Comment:
Why does it make sense to log a `\n` here but not everywhere else?

Reply:
So I had intended some log messages to be closer together if they're related (e.g. "Starting wait" and "Wait complete") and others use newlines for spacing. Since error logs aren't normally supposed to happen, I added newlines here. I'll add them to the other error logs too.